packages/adp-tooling/src/base/prompt.ts

```diff
@@ -16,6 +16,8 @@ export type PromptDefaults = {
     ft?: boolean;
     package?: string;
     transport?: string;
+    username?: string;
```

Comment:
Please reuse https://github.com/SAP/open-ux-tools/blob/main/packages/adp-tooling/src/base/prompt.ts#L115 for connecting to a system and authenticating. We need more than just basic auth support.

Reply:
Fix with: b71f9c20e861416fd5f2fb9fc05dfd336700dafb